src/core/lib/slice/slice_intern.cc

```diff
@@ -56,7 +51,7 @@ typedef struct slice_shard {
 } slice_shard;
 
 /* hash seed: decided at initialization time */
-static uint32_t g_hash_seed;
+uint32_t g_hash_seed;
```

Comment:
Great catch.

Reply:
Yeah, needed the linkage change so I could inline the hash method for interned slices.